keyboards/keycapsss/kimiko/rules.mk

```diff
@@ -1,15 +1 @@
-# Build Options
-#   change yes to no to disable
-#
-BOOTMAGIC_ENABLE = no       # Enable Bootmagic Lite
-MOUSEKEY_ENABLE = yes       # Mouse keys
-EXTRAKEY_ENABLE = yes       # Audio control and System control
-CONSOLE_ENABLE = no         # Console for debug
-COMMAND_ENABLE = no         # Commands for debug and configuration
-NKRO_ENABLE = no            # Enable N-Key Rollover
-BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
-RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
-AUDIO_ENABLE = no           # Audio output
-SPLIT_KEYBOARD = yes
-
-DEFAULT_FOLDER = keycapsss/kimiko/rev1
+# File is intentionally left blank
```

Comment:
removing this file should will fix linting error, which in turn will correct CI Build failure.

Reply:
linting is failing because a blank `rules.mk` exists in `keycapsss/kimiko' directory which tells QMK the directory is a keyboard, which isn't intended.  
I'd advice removing this file for this PR, and set DEFAULT_FOLDER in the respectively named "rev2" PR

Reply:
Ah thanks. I removed the rules.mk here and set rev2 as default here https://github.com/qmk/qmk_firmware/pull/21719#pullrequestreview-1619585111

Reply:
good stuff, the remaining CI Build failure is OLED_FONT_H file path in `config.h` file of `oraji3` keymap directory 😉